vignettes/functional_data.Rmd

```diff
@@ -80,49 +82,49 @@ The data is provided in the following structure:
 str(fuelSubset)
 ```
 
-* __heatan__ is the target variable, in this case a numeric value.
+* __heatan__ is the target variable, in this case, a numeric value.
 * __h2o__ is an additional scalar variable.
-* __NIR__ and __UVVIS__ are matrices containing the curve data. Each column corresponds to a single time point the data was sampled at. Each row indicates a single curve. 
-__NIR__ was measured at $231$ time points, while __UVVIS__ was measured at $129$ time points.
-* __nir.lambda__ and __uvvis.lambda__ are numeric vectors of length $231$ and $129$ indicate the time points the data was measured at. 
-Each entry corresponds to one column of __NIR__ and __UVVIS__ respectively. For now we ignore this additional information in mlr.
+* __NIR__ and __UVVIS__ are matrices containing the curve data. Each column corresponds to a single wavelength the data was sampled at.
+ Each row indicates a single curve. 
+__NIR__ was measured at $231$ wavelength points, while __UVVIS__ was measured at $129$ wavelength points.
```

Comment:
I find it better to have numbers not formatted as LaTeX numbers. We should just write 129 and 231.
This applies to all occurrences then.